pydatalab/pydatalab/models/people.py

```diff
@@ -62,6 +62,27 @@ def add_missing_verification(cls, v):
         return v
 
 
+class DisplayName(str):
+    """
+    A constrained string less than 150 characters long
+    that cannot be empty and should contain only alphabetic characters and spaces.
+    """
+
+    max_length = 150
+
+    def __new__(cls, value):
+        if not value.strip():
+            raise ValueError("Display name cannot be empty.")
+
+        if not all(char.isalpha() or char.isspace() for char in value):
```

Comment:
We need to allow hyphens and apostrophes through in names too (maybe others?) I actually don't know how e.g., a name with Chinese characters would be best handled here

Reply:
See my comment above, I think this is a can of worms that we should try to close again...